frontend/app/components/input-checkboxes.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ComponentProps, ReactNode } from 'react';
+
+import { InputCheckbox } from './input-checkbox';
+import { InputError } from './input-error';
+import { InputHelp } from './input-help';
+import { InputLegend } from '~/components/input-legend';
+import { cn } from '~/utils/tw-utils';
+
+export interface InputCheckboxesProps {
+  errorMessage?: string;
+  helpMessagePrimary?: React.ReactNode;
+  helpMessagePrimaryClassName?: string;
+  helpMessageSecondary?: React.ReactNode;
+  helpMessageSecondaryClassName?: string;
+  id: string;
+  options: Omit<ComponentProps<typeof InputCheckbox>, 'aria-describedby' | 'aria-errormessage' | 'aria-invalid' | 'aria-required' | 'id' | 'name' | 'required'>[];
+  legend: ReactNode;
+  name: string;
+  required?: boolean;
+}
+
+const InputCheckboxes = ({ errorMessage, helpMessagePrimary, helpMessagePrimaryClassName, helpMessageSecondary, helpMessageSecondaryClassName, id, legend, name, options, required }: InputCheckboxesProps) => {
```

Comment:
Our convention (and indeed Remix's convention) is to use `export function InputCheckboxes(..) { .. }` instead of exporting arrow functions.

Reply:
Implemented in commit dec2ceef9218bed7957453837c8f49c7cb1d2692.